.gitignore

```diff
@@ -1,5 +1,7 @@
+node_modules
 /lib/
 /es/
-/node_modules/
-/built/
+/esm/
+/test/babel/
+/test/typescript/
```

Comment:
Tests, from both transpilers, now build into these subdirectories so that they can be compared.